### DISCUSSION TITLE
Push notifications

### DISCUSSION THREAD

**Discussion by pwFoo** [directus/directus#4619] [Archive]
Hi,
searching for a simple way to trigger push notifications to clients for example with webpush api (Android, iOS, browsers) / client side apps (gonify server with android client?)...

Anyone played with gonify, fcm or other solutions and would it be possible to trigger by directus webhooks?
I would be interested in howtos or some more information about directus with push notifications, because I'm new to that topic.

Regards

**Comment by rijkvanzanten** [comment]
Directus doesn't do any push notifications itself. That being said, you should be able to use [a Hook](https://docs.directus.io/guides/api-hooks/#custom-api-hooks) or a webhook to trigger a push notification externally.

**Reply by pwFoo** [reply]
Hi @rijkvanzanten,
thanks. I think webhook should be the easiest way to do that, but I don't know how to build such a push notification feature with all the parts (client register, save registration, trigger push based on content and registered user...).

I was hoping someone had already dealt with this topic and had examples to share :)

**Reply by Jeyakumarrathnaselvi** [reply]
> Hi @rijkvanzanten, thanks. I think webhook should be the easiest way to do that, but I don't know how to build such a push notification feature with all the parts (client register, save registration, trigger push based on content and registered user...).
> 
> I was hoping someone had already dealt with this topic and had examples to share :)

@ have you found any method to implement push notification, i am struggling with it for long

**Reply by fa-briz-io** [reply]
![Screenshot 2022-11-05 at 10 56 28](https://user-images.githubusercontent.com/7761958/200114277-bbc31e4a-da9a-410f-99e6-234d81ce41c9.jpg)
It is actually pretty simple (according to me) in Directus, I am using flow and webhook to achieve this. I hook into a event and manipulate what I need and then send a POST to [Firebase Cloud Messaging](https://firebase.google.com/docs/cloud-messaging). The hard part to me was implementing it into the app (I'm using Flutter).

**Comment by anjana096** [comment]
@Jeyakumarrathnaselvi do you found any solution for send push notification in directus?
